src/tpt_reports/tpt_reports.py

```diff
@@ -154,10 +155,20 @@ def main() -> None:
 
     try:
         validated_args: Dict[str, Any] = schema.validate(args)
+        validators.domain(validated_args["DOMAIN_TESTED"])
+
     except SchemaError as err:
         # Exit because one or more of the arguments were invalid
         print(err, file=sys.stderr)
         sys.exit(1)
+    except ValueError as err:
+        # Exit due to invalid value supplied
+        print(err, file=sys.stderr)
+        sys.exit(1)
+    except OSError as err:
+        # Exit if an invalid file path is provided
+        print(err, file=sys.stderr)
+        sys.exit(1)
```

Comment:
This PR should also include one or more new tests that include values for `DOMAIN_TESTED` that trigger these types of errors in various ways.

You should probably also add a test that include values for `DOMAIN_TESTED` that validate successfully (beyond what you changed in `test_log_levels()`).

Reply:
Test case is added in: 0d6bcf1aad0d5f680c227cfe533c454e9023ae9c

Reply:
It's a pretty bare-bones test case, but it gets the job done.  Can you please modify the assertion text to mention return code 2?  That will be helpful if this test ever fails.

It would also be nice to see a similar test for the `OSError` exception (`return_code == 3`).

Reply:
I believe you are looking at an outdated commit. We removed OSError because we moved some validation to Issue: https://github.com/cisagov/tpt-reports/issues/39 and the remaining validator would never throw an instance of OSError. Please take a look at the latest file changes.